cds_books/migrator/api.py

```diff
@@ -186,18 +201,18 @@ def create_multipart_volumes(pid, multipart_legacy_recid, migration_volumes):
     first_volume = next(volume_numbers)
     first = Document.get_record_by_pid(pid)
     if 'title' in volumes[first_volume]:
-        first['title']['title'] = volumes[first_volume]['title']
+        first['title'] = volumes[first_volume]['title']
         first['volume'] = first_volume
     first['_migration']['multipart_legacy_recid'] = multipart_legacy_recid
-    if 'legacy_recid' in first:
-        del first['legacy_recid']
+    # if 'legacy_recid' in first:
```

Comment:
no commented code

Reply:
same explanation as the previous comment